wallet-mobile-app-centbit/Components/HomeFourthCom.js

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react'
+import { View, Image, Text } from 'react-native'
+import { COLORS } from '../Constants/colors';
+import { HOMEIMAGSPOTIFY } from '../Constants/file';
+
+export default function HomeFourthComponent(){
+     return(
+          <View>
+               <View style={{
+                    backgroundColor: COLORS.colorblack,
+                    marginLeft: 20,
+                    marginRight: 20,
+                    height: 80,
+                    borderRadius: 20,
+                    marginBottom: 10,
+                    justifyContent: 'space-between',
+                    flexDirection: 'row',
+                    paddingLeft: 15,
+                    paddingEnd: 15,
+                    paddingTop: 18
+               }}>
+                    <View>
+                         <Image source={require('../assets/spotify.webp')} style={{ height: '60%', aspectRatio: 1, resizeMode: 'stretch'}}/>
```

Comment:
try as much as possible to avoid hardcoding height, width etc, if we are to switch or build for different screen sizes, the % will break at some point